packages/react/src/Timeline/Timeline.tsx

```diff
@@ -0,0 +1,40 @@
+import React, {PropsWithChildren} from 'react'
+import clsx from 'clsx'
+import styles from './Timeline.module.css'
+import {BaseProps} from '../component-helpers'
+import {useAnimation} from '../animation'
+
+export type TimelineProps = BaseProps<HTMLUListElement>
+
+const _TimelineRoot = ({animate, className, children, ...rest}: PropsWithChildren<TimelineProps>) => {
+  const {classes: animationClasses, styles: animationInlineStyles} = useAnimation(animate)
+
+  const timelineClassName = clsx(animationClasses, styles.Timeline, className)
+
+  return (
+    // TODO add ...stye
+    <ul className={timelineClassName} style={{...animationInlineStyles}} {...rest}>
+      {children}
+    </ul>
+  )
+}
+
+export type TimelineItemProps = BaseProps<HTMLLIElement>
+
+const Item = ({className, children, ...rest}: PropsWithChildren<TimelineItemProps>) => {
+  const itemClassName = clsx(styles['Timeline-item'], className)
```

Comment:
We're admittedly a bit inconsistent around it's application, but historically we've tried to follow BEM where possible. Could you change this to `Timeline__item` instead?